### PR TITLE
[mainnet] added paraswap v5

### PR DESCRIPTION
**Pull request by bhavik-m**
- [x] Added paraswap on mainnet + minor fixes
 - [x] comment deployed addresses
-  [x] add to dsa-connect [here](https://github.com/Instadapp/dsa-connect/pull/119)

### CODE REVIEW

**Review by pradyuman-verma — APPROVED** [review]

**Comment by bhavik-m** [comment]
paraswap test cases are passing
Screenshot (437)

File: test/mainnet/paraswap/paraswap.test.ts [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    //     method: "hardhat_reset",
+    //     params: [
+    //         {
+    //             forking: {
+    //                 jsonRpcUrl: hre.config.networks.hardhat.forking.url,
+    //                 blockNumber: 13300000,
+    //             },
+    //         },
+    //     ],
+    // });
```
Comment (thrilok209) [inline test/mainnet/paraswap/paraswap.test.ts]: @bhavik-m Why is this commented?

Comment (bhavik-m) [reply test/mainnet/paraswap/paraswap.test.ts]: I commented this since i was only running a single test. we needed this when we run multiple test cases

**Comment by bhavik-m** [comment]
## Deployed on mainnet:

**Name:** `PARASWAP-V5-A`
**Addr:** [0x888d85fe4eafe3d462f11faa1b22c0d860be8f3f](https://etherscan.io/address/0x888d85fe4eafe3d462f11faa1b22c0d860be8f3f)
**Transaction**: [link](https://etherscan.io/tx/0x4489baa978c1e1d55de15c5b78ed6a758c6794838500c6294ba682fa79ca5797)

**Pull request merged**